f8a_notification/entryscript.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © 2018 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Geetika Batra <gbatra@redhat.com>
+#
+
+"""Class to create cron script."""
+
+
+def run():
```

Comment:
Missing docstring :D

Reply:
This is a temporary function provided to test if the script works with docker and openshift. @yzainee's changes will override this once this PR is merged. :)

Reply:
Ack, please ignore my comment then :wink: